src/prefixmaps/data/bioportal.curated.yaml

```diff
@@ -0,0 +1,509 @@
+name: bioportal
+prefixes:
```

Comment:
would be cool if we could converge on the "extended prefix map" schema

Reply:
Would be nice, yes - but for now, this is still parsed to canonical/prefix_alias/namespace_alias so I'll save adopting the extended schema for a near-future PR